### DISCUSSION TITLE
Advanced Tooltips

### DISCUSSION THREAD

**Discussion by s6an** [BedrockTweaks/Files#237] [Ideas]
A feature that would be both useful and helpful is advanced tooltips that is currently only on the java version of the game.

**Comment by SquatchHunter** [comment]
can you show a screen shot of what you would like it to look like?

**Reply by s6an** [reply]
![40d13-16318258379170-800](https://github.com/user-attachments/assets/0f208aac-dcf8-46ce-8f66-cccc9149916e)
Just basically showing the durability on tools and armor and just gives more info on items
